src/materialized/src/http/prof.rs

```diff
@@ -262,6 +262,17 @@ mod enabled {
                     .body(Body::from(s))
                     .unwrap())
             }
+            "dump_stats" => {
+                let mut borrow = prof_ctl.lock().await;
+                let s = borrow.dump_stats()?;
+                Ok(Response::builder()
+                    .header(
+                        header::CONTENT_DISPOSITION,
+                        "attachment; filename=\"jestats.txt\"",
+                    )
```

Comment:
No strong preference, but seems like it might be nicer (assuming these stats are human readable) to just render the text file directly? And then users can save the file or `curl` it to disk if they like.

Reply:
Makes sense, changed it to render the text in the browser. Thanks!